src/scenarios/shared/runner.py

```diff
@@ -637,7 +637,7 @@ def run(self):
             getLogger().info("Completed install.")
 
             allResults = []
-            timeToFirstDrawEventEndDateTime = datetime.now() # This is used to keep track of the latest time to draw end event, we use this to calculate time to draw and also as a reference point for the next iteration log time.
+            timeToFirstDrawEventEndDateTime = datetime.now() + timedelta(minutes=-10) # This is used to keep track of the latest time to draw end event, we use this to calculate time to draw and also as a reference point for the next iteration log time.
```

Comment:
Do we ever intend to use the value that is ten minutes in the past, or do we always have a warmup iteration?

Reply:
We should never have a reason to use a value in the actual past, just potentially in the past compared to the hosting computer if the iPhone and computer times get out of sync. For the warmup iteration, I could see a style of device startup where we uninstall and reinstall between every run, but until we have that I am pretty sure we will have a warmup run, and even if we add it we could use a different flow.